src/librustc_save_analysis/sig.rs

```diff
@@ -145,49 +153,52 @@ fn text_sig(text: String) -> Signature {
     Signature { text, defs: vec![], refs: vec![] }
 }
 
-fn push_extern(text: &mut String, ext: Extern) {
+/*fn push_extern(text: &mut String, ext: Extern) {
```

Comment:
You forgot to remove this dead code.

Reply:
Nice catch! I scanned the code last time and somehow missed that and thought it's resolved, let's wait with this for now

Reply:
(hopefully bors works in the review comments)

@bors r-

Reply:
Oh sorry fixing it now!